cloud_connectors/azure/mqtt_connector/azure_function/tests/MQTTConnectorAzureFunction.Tests.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license.
 // SPDX-License-Identifier: MIT
```

Comment:
What about the Microsoft part of the license?  It seems to be missing.

Reply:
it's already there on line 1